src/UmikoBot.cpp

```diff
@@ -811,4 +719,4 @@ void UmikoBot::GetGuildMemberInformation(snowflake_t guild, snowflake_t after)
 		listGuildMembers(guild, 1000).then(processMembers);
 	else
 		listGuildMembers(guild, 1000, after).then(processMembers);
-}
+}
```

Comment:
It's unfortunately difficult to see what's actually changed, and what's just been reformatted. I would love to see a diff that's actually representative of what's functionally different.

Reply:
(In case it's not clear, I'm talking about this whole file UmikoBot.cpp)

Reply:
Changes reverted!